File: api/internal/model/stats.go
package model

import "time"

type UserActivity struct {
	Id         int64          `json:"id"`
	UserId     int64          `json:"user_id"`
	Action     string         `json:"action"`
	Timestamp  time.Time      `json:"timestamp"`
	ActionData map[string]any `json:"action_data"`
}

type DailyStat struct {
	DAU  int       `json:"dau"`
	Date time.Time `json:"date"`
}

File: bot/app/model/stats.py
from datetime import datetime
from dataclasses import dataclass

@dataclass
class UserActivity:
    id: int = 0
    user_id: int = 0
    action: str = ""
    time: datetime = datetime.now()

@dataclass
class DailyStat:
    dau: int = 0
    date: datetime = datetime.now()